### PR TITLE
chore: refactory

### PR DESCRIPTION
**Pull request by RafilxTenfen**
- Move `PrivateSigner` to specific package
- Created vars for constant used Mod accounts in appparams

### CODE REVIEW

File: testutil/signer/private.go [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+// SetupTestPrivSigner sets up a PrivSigner for testing
+func SetupTestPrivSigner() (*signer.PrivSigner, error) {
+	// Create a temporary node directory
+	nodeDir, err := os.MkdirTemp("", "tmp-signer")
+	if err != nil {
+		return nil, err
+	}
+	defer func() {
+		_ = os.RemoveAll(nodeDir)
+	}()
```
Comment (Lazar955) [inline testutil/signer/private.go:23]:
```suggestion
	nodeDir := t.TempDir()
```

Comment (Lazar955) [reply testutil/signer/private.go:23]: t.TempDir() internally calls t.CleanUp, so a bit cleaner

**Review by Lazar955 — APPROVED** [review]
lgtm

File: testutil/signer/private.go (hunk 1 above)
Comment (RafilxTenfen) [reply testutil/signer/private.go:23]: problem is, I don't have the `*testing.T` there

**Pull request merged**